package.json

```diff
@@ -1,150 +1,16 @@
 {
-  "name": "ember-bootstrap",
-  "version": "6.1.0",
-  "description": "Bootstrap components for Ember.js",
-  "keywords": [
-    "ember-addon",
-    "bootstrap"
-  ],
-  "homepage": "https://www.ember-bootstrap.com/",
-  "bugs": {
-    "url": "https://github.com/ember-bootstrap/ember-bootstrap/issues"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/ember-bootstrap/ember-bootstrap"
-  },
-  "license": "MIT",
-  "author": "Simon Ihmig <ihmig@kaliber5.de>",
-  "directories": {
-    "doc": "docs",
-    "test": "tests"
-  },
+  "private": true,
   "scripts": {
-    "build": "ember build --environment=production",
-    "changelog": "lerna-changelog",
-    "docs:build": "gulp docs:build",
-    "docs:serve": "gulp docs:serve",
-    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
-    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
-    "lint:hbs": "ember-template-lint .",
-    "lint:hbs:fix": "ember-template-lint . --fix",
-    "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
-    "nodetest": "mocha node-tests --recursive",
-    "start": "ember serve",
-    "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each",
-    "prepare": "husky install"
-  },
-  "lint-staged": {
-    "*.js": "eslint --cache --fix"
-  },
-  "dependencies": {
-    "@ember/render-modifiers": "^2.0.0",
-    "@embroider/macros": "^1.0.0",
-    "@embroider/util": "^1.0.0",
-    "broccoli-debug": "^0.6.3",
-    "broccoli-funnel": "^3.0.8",
-    "broccoli-merge-trees": "^4.2.0",
-    "chalk": "^4.1.1",
-    "ember-auto-import": "^2.6.3",
-    "ember-cli-babel": "^8.0.0",
-    "ember-cli-build-config-editor": "^0.5.1",
-    "ember-cli-htmlbars": "^6.2.0",
-    "ember-cli-version-checker": "^5.1.2",
-    "ember-concurrency": "^2.1.2 || ^3.0.0",
-    "ember-element-helper": "^0.6.0 || ^0.7.0 || ^0.8.0",
-    "ember-focus-trap": "^1.0.0",
-    "ember-on-helper": "^0.1.0",
-    "ember-popper-modifier": "^2.0.0 || ^3.0.0 || ^4.0.0",
-    "ember-ref-bucket": "^4.0.0 || ^5.0.0",
-    "ember-render-helpers": "^0.2.0",
-    "ember-style-modifier": "^0.8.0 || ^1.0.0 || ^2.0.0 || ^3.0.0",
-    "findup-sync": "^5.0.0",
-    "fs-extra": "^11.0.0",
-    "resolve": "^1.18.1",
-    "silent-error": "^1.0.1",
-    "tracked-toolbox": "^1.2.3 || ^2.0.0"
+    "prepare": "husky install",
+    "test": "pnpm --filter 'test-app' test",
+    "test:ember": "pnpm --filter 'test-app' test:ember"
```

Comment:
Should we add `lint` and `lint:fix` scripts as well, which run on all three projects? Might be helpful to quickly resolve linting issues.